src/com/palmergames/bukkit/towny/tasks/HourlyTimerTask.java

```diff
@@ -45,7 +49,7 @@ public void run() {
 	}
 
 	/*
-	 * Reduce the number of hours jailed residents are jailed for.
+	 * Reduce the number of hours jailed residents are jailed for and remove hourly jail fee from town bank if possible
 	 */
 	private void decrementJailedHours() {
```

Comment:
This method should be renamed to be more descriptive, ie: decrementJailedHoursAndIncurJailFees().

Reply:
Method has been renamed in new commit to aforementioned decrementJailedHoursAndIncurJailFees() to better describe its utility.